install/kubernetes/cilium/values.yaml

```diff
@@ -173,7 +173,7 @@ azure:
   # userAssignedIdentityID: 00000000-0000-0000-0000-000000000000
 
 # TODO: Add documentation
-bandwidthManager: false
+bandwidthManager: true
```

Comment:
Would this be the time to add documentation?

Reply:
Why not, this commit is a revert but I can add a new one to comment the value. Thanks!